adm/src/main.rs

```diff
@@ -17,14 +17,8 @@
 
 #[macro_use]
 extern crate clap;
-extern crate libc;
```

Comment:
Could you move these removals to another commit?

Reply:
Moved to separate commit